CONTRIBUTING.md

```diff
@@ -0,0 +1,154 @@
+## How to contribute to SST-Macro
+
+#### **Source Code Repository**
+
+* SST-Macro is the repository for the Sandia developed SST Simulation Macro.  The repository is hosted on [GitHub](https://github.com).
```

Comment:
"Sandia-developed macroscale components for the Structural Simulation Toolkit (SST)."